app/routes/cdo/create/select-address.js

```diff
@@ -51,6 +52,14 @@ module.exports = [
 
         setAddress(request, address)
 
+        const enforcementDetails = getEnforcementDetails(request) || {}
+        const policeForce = await lookupPoliceForceByPostcode(address.postcode)
+
+        if (policeForce) {
+          enforcementDetails.policeForce = policeForce.id
```

Comment:
We may need to rethink this at some point as I think the API should handle the lookup part rather than passing the force id returned by the API.

Appreciate it isn't possible here at the moment though due to the hyphenated police force.